src/scripts.js

```diff
@@ -502,4 +503,11 @@ function cookRecipe() {
    removeIngredientsFromPantry().forEach(element => {
         updatePantry(element)
     })
+}
+
+function fixEnter(event){
+    if(event.keyCode === 13){
```

Comment:
it also looks like keyCode is deprecated, should we use event.key? https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
All code has been adjusted to use event.key for consistency